python/UniverseWrapper.cpp

```diff
@@ -146,6 +146,9 @@ namespace {
     const Ship::PartMeterMap&
                             (Ship::*ShipPartMeters)(void) const =                               &Ship::PartMeters;
 
+    const ShipDesign*       GetGenericShipDesign(const Universe& universe, const std::string& name)
+    { return universe.GetGenericShipDesign(name); }
```

Comment:
Just out of curiosity: why still stick to a wrapper function instead of just exposing the member function `GetGenericShipDesign` of the `Universe` class?

I'm asking because I'm confused about all those different ways member functions get exposed to Python in FO, and I wonder if you actually know more about this than me. There are member functions that get directly exposed (e.g. `Universe::SystemHasVisibleStarlanes`), others have these wrappers, and still others use this `make_function` function.

Which is used for what and why? The Boost Python documentation isn't partucularly enlightening. In some cases I can guess at the reasons, but others still leave me clueless. In this case (`Universe::GetGenericShipDesign`) I would have just exposed the member function directly...